src/components/AdoptionForm.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import ErrorList from "./ErrorList";
+import validateForm from "../functions/validateForm";
+import postData from "../functions/postData";
+
+const AdoptionForm = props => {
+  const POST_API_PATH = "/api/v1/applicants";
```

Comment:
Asking for a little clarification here. You are declaring an immutable global variable here. Though it should be fine to use normal camel case here. When using it did it not work till it was immutable? If you do intend to use it as that type of Global, it is normally convention in React to save it as a file and import it anywhere you need to use it, since the imports at the top are all accessed as if they are globals.

Reply:
not supposed to be global just habit from using with with .env files and APIs in general. I'll change this